### PR TITLE
Forest

### PR DESCRIPTION
**Pull request by andrewgryan**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/forest/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/informatics-lab/forest/archive/v{{ version }}.tar.gz
+  sha256: d06a17ec394ed737fe265d5c163a4e536be3ae20f10443968ba7a04457b710e0
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (scopatz) [inline recipes/forest/meta.yaml]: Can this be `noarch: python`?

Comment (andrewgryan) [reply recipes/forest/meta.yaml]: I'm not sure, does noarch: python mean it's just pure python? In which case, yes it can be

Comment (scopatz) [reply recipes/forest/meta.yaml]: Yep that is what that means

Comment (andrewgryan) [reply recipes/forest/meta.yaml]: Great, in that case I'll see if I can add it to the pull request. I'm in an airport at the moment with only a smart phone to develop on but I'll see what I can do

**Comment by andrewgryan** [comment]
Hmm, that didn't seem to work...

**Comment by scopatz** [comment]
That worked fine.  noarch python only needs to work on linux

**Comment by andrewgryan** [comment]
That's a relief, I put it back in for safe measure. Let me know if there's anything else I can do. About to board now, thanks for your help!

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**